From: "Eric W. Biederman" <ebiederm@xmission.com>
Subject: MAINTAINERS: remove Eric
Date: Mon, 17 Mar 2025 08:47:30 -0500

Unfortunately I no longer have time to meaningfully take part in the
linux kernel development.

Link: https://lkml.kernel.org/r/87r02v7or1.fsf@email.froward.int.ebiederm.org
Signed-off-by: Eric W. Biederman <ebiederm@xmission.com>
Cc: Kees Cook <keescook@chromium.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 MAINTAINERS |    3 ---
 1 file changed, 3 deletions(-)

--- a/MAINTAINERS~maintainers-remove-myself
+++ a/MAINTAINERS
@@ -8643,7 +8643,6 @@ F:	rust/kernel/net/phy/reg.rs
 
 EXEC & BINFMT API, ELF
 M:	Kees Cook <kees@kernel.org>
-R:	Eric Biederman <ebiederm@xmission.com>
 L:	linux-mm@kvack.org
 S:	Supported
 T:	git git://git.kernel.org/pub/scm/linux/kernel/git/kees/linux.git for-next/execve
@@ -12824,9 +12823,7 @@ F:	fs/kernfs/
 F:	include/linux/kernfs.h
 
 KEXEC
-M:	Eric Biederman <ebiederm@xmission.com>
 L:	kexec@lists.infradead.org
-S:	Maintained
 W:	http://kernel.org/pub/linux/utils/kernel/kexec/
 F:	include/linux/kexec.h
 F:	include/uapi/linux/kexec.h
_
